--- redhat-menus-8.9.10/desktop-files/redhat-evolution-tasks.desktop.in.evolution	2005-10-28 13:31:58.000000000 -0400
+++ redhat-menus-8.9.10/desktop-files/redhat-evolution-tasks.desktop.in	2007-06-07 12:15:55.000000000 -0400
@@ -8,6 +8,8 @@ Type=Application
 Encoding=UTF-8
 Categories=X-Red-Hat-Base;X-Red-Hat-Base-Only;Office;Application;
 StartupNotify=True
-X-GNOME-Bugzilla-Bugzilla=Ximian
+X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
-X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.4;evolution-exchange-storage;evolution-alarm-notify;
+X-GNOME-Bugzilla-Version=2.12
+X-GNOME-Bugzilla-Component=Tasks
+X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
--- redhat-menus-8.9.10/desktop-files/redhat-evolution-calendar.desktop.in.evolution	2005-10-28 13:31:58.000000000 -0400
+++ redhat-menus-8.9.10/desktop-files/redhat-evolution-calendar.desktop.in	2007-06-07 12:15:32.000000000 -0400
@@ -8,6 +8,8 @@ Type=Application
 Encoding=UTF-8
 Categories=X-Red-Hat-Base;X-Red-Hat-Base-Only;Office;Application;
 StartupNotify=True
-X-GNOME-Bugzilla-Bugzilla=Ximian
+X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
-X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.4;evolution-exchange-storage;evolution-alarm-notify;
+X-GNOME-Bugzilla-Version=2.12
+X-GNOME-Bugzilla-Component=Calendar
+X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
--- redhat-menus-8.9.10/desktop-files/redhat-evolution-contacts.desktop.in.evolution	2005-10-28 13:31:58.000000000 -0400
+++ redhat-menus-8.9.10/desktop-files/redhat-evolution-contacts.desktop.in	2007-06-07 12:15:40.000000000 -0400
@@ -8,6 +8,8 @@ Type=Application
 Encoding=UTF-8
 Categories=X-Red-Hat-Base;X-Red-Hat-Base-Only;Network;Application;
 StartupNotify=True
-X-GNOME-Bugzilla-Bugzilla=Ximian
+X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
-X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.4;evolution-exchange-storage;evolution-alarm-notify;
+X-GNOME-Bugzilla-Version=2.12
+X-GNOME-Bugzilla-Component=Contacts
+X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
--- redhat-menus-8.9.10/desktop-files/redhat-evolution-mail.desktop.in.evolution	2005-10-28 13:31:58.000000000 -0400
+++ redhat-menus-8.9.10/desktop-files/redhat-evolution-mail.desktop.in	2007-06-07 12:15:47.000000000 -0400
@@ -8,6 +8,8 @@ Type=Application
 Encoding=UTF-8
 Categories=X-Red-Hat-Base;X-Red-Hat-Base-Only;Network;Application;
 StartupNotify=True
-X-GNOME-Bugzilla-Bugzilla=Ximian
+X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
-X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.4;evolution-exchange-storage;evolution-alarm-notify;
+X-GNOME-Bugzilla-Version=2.12
+X-GNOME-Bugzilla-Component=Mailer
+X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
