diff -up redhat-menus-10.0.1/menus/preferences.menu.categories redhat-menus-10.0.1/menus/preferences.menu
--- redhat-menus-10.0.1/menus/preferences.menu.categories	2009-02-25 00:10:51.612815552 -0500
+++ redhat-menus-10.0.1/menus/preferences.menu	2009-02-25 00:11:06.782816607 -0500
@@ -9,9 +9,6 @@
   <DefaultAppDirs/>
   <DefaultDirectoryDirs/>
 
-  <!-- Read in overrides and child menus from preferences-merged/ -->
-  <DefaultMergeDirs/>
-
   <!-- Stuff in the toplevel -->
   <Include>
     <And>
@@ -27,4 +24,7 @@
     <Filename>gnomecc.desktop</Filename>
   </Exclude>
 
+  <!-- Read in overrides and child menus from preferences-merged/ -->
+  <DefaultMergeDirs/>
+
 </Menu>     <!-- End Prefs -->
