diff -up redhat-menus-8.9.11/desktop-files/redhat-evolution-tasks.desktop.in.evolution redhat-menus-8.9.11/desktop-files/redhat-evolution-tasks.desktop.in
--- redhat-menus-8.9.11/desktop-files/redhat-evolution-tasks.desktop.in.evolution	2008-01-05 18:47:35.000000000 -0500
+++ redhat-menus-8.9.11/desktop-files/redhat-evolution-tasks.desktop.in	2008-01-05 18:48:18.000000000 -0500
@@ -11,5 +11,5 @@ StartupNotify=True
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
 X-GNOME-Bugzilla-Version=2.12
-X-GNOME-Bugzilla-Component=Tasks
+X-GNOME-Bugzilla-Component=BugBuddyBugs
 X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
diff -up redhat-menus-8.9.11/desktop-files/redhat-evolution-calendar.desktop.in.evolution redhat-menus-8.9.11/desktop-files/redhat-evolution-calendar.desktop.in
--- redhat-menus-8.9.11/desktop-files/redhat-evolution-calendar.desktop.in.evolution	2008-01-05 18:47:16.000000000 -0500
+++ redhat-menus-8.9.11/desktop-files/redhat-evolution-calendar.desktop.in	2008-01-05 18:47:51.000000000 -0500
@@ -11,5 +11,5 @@ StartupNotify=True
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
 X-GNOME-Bugzilla-Version=2.12
-X-GNOME-Bugzilla-Component=Calendar
+X-GNOME-Bugzilla-Component=BugBuddyBugs
 X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
diff -up redhat-menus-8.9.11/desktop-files/redhat-evolution-contacts.desktop.in.evolution redhat-menus-8.9.11/desktop-files/redhat-evolution-contacts.desktop.in
--- redhat-menus-8.9.11/desktop-files/redhat-evolution-contacts.desktop.in.evolution	2008-01-05 18:47:25.000000000 -0500
+++ redhat-menus-8.9.11/desktop-files/redhat-evolution-contacts.desktop.in	2008-01-05 18:47:59.000000000 -0500
@@ -11,5 +11,5 @@ StartupNotify=True
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
 X-GNOME-Bugzilla-Version=2.12
-X-GNOME-Bugzilla-Component=Contacts
+X-GNOME-Bugzilla-Component=BugBuddyBugs
 X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
diff -up redhat-menus-8.9.11/desktop-files/redhat-evolution-mail.desktop.in.evolution redhat-menus-8.9.11/desktop-files/redhat-evolution-mail.desktop.in
--- redhat-menus-8.9.11/desktop-files/redhat-evolution-mail.desktop.in.evolution	2008-01-05 18:47:31.000000000 -0500
+++ redhat-menus-8.9.11/desktop-files/redhat-evolution-mail.desktop.in	2008-01-05 18:48:08.000000000 -0500
@@ -11,5 +11,5 @@ StartupNotify=True
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=Evolution
 X-GNOME-Bugzilla-Version=2.12
-X-GNOME-Bugzilla-Component=Mailer
+X-GNOME-Bugzilla-Component=BugBuddyBugs
 X-GNOME-Bugzilla-OtherBinaries=evolution-data-server-1.12;evolution-exchange-storage;evolution-alarm-notify;
