--- redhat-menus-8.9.10/menus/applications.menu.hide-screensavers	2007-06-26 15:38:51.000000000 -0400
+++ redhat-menus-8.9.10/menus/applications.menu	2007-06-26 15:39:10.000000000 -0400
@@ -170,6 +170,7 @@
         <Not><Category>Monitor</Category></Not>
         <Not><Category>SystemSetup</Category></Not>
         <Not><Category>X-Red-Hat-ServerConfig</Category></Not>
+        <Not><Category>Screensaver</Category></Not>
       </And>
     </Include>
   </Menu> <!-- End Other -->
