--- redhat-menus-7.8.9/menus/applications.menu.cc-shell	2007-02-08 23:37:21.000000000 -0500
+++ redhat-menus-7.8.9/menus/applications.menu	2007-02-08 23:45:21.000000000 -0500
@@ -119,7 +119,7 @@
   <!-- System Tools-->
   <Menu>
     <Name>System Tools</Name>
-    <Directory>System.directory</Directory>
+    <Directory>System-Tools.directory</Directory>
     <Include>
       <And>
         <Category>System</Category>
--- redhat-menus-7.8.9/menus/preferences.menu.cc-shell	2005-11-22 01:38:46.000000000 -0500
+++ redhat-menus-7.8.9/menus/preferences.menu	2007-02-08 23:36:55.000000000 -0500
@@ -3,78 +3,118 @@
 
 <Menu>
   <Name>Preferences</Name>
-  <Directory>Settings.directory</Directory>
-
-  <!-- Scan legacy dirs first, as later items take priority -->
-  <LegacyDir>/usr/share/gnome/apps</LegacyDir>
-  <LegacyDir>/etc/X11/applnk</LegacyDir>
+  <Directory>Preferences.directory</Directory>
 
   <!-- Read standard .directory and .desktop file locations -->
   <DefaultAppDirs/>
   <DefaultDirectoryDirs/>
 
-  <!-- Read in overrides and child menus from preferences.d -->
+  <!-- Read in overrides and child menus from preferences-merged/ -->
   <DefaultMergeDirs/>
 
+  <!-- Sort the control center categories -->
+  <Layout>
+    <Menuname>Personal</Menuname>
+    <Menuname>Look and Feel</Menuname>
+    <Menuname>Internet and Network</Menuname>
+    <Menuname>Hardware</Menuname>
+    <Menuname>System</Menuname>
+    <Merge type="all" />
+  </Layout>
+  
   <!-- Stuff in the toplevel -->
   <Include>
     <And>
       <Category>Settings</Category>
-      <Category>X-Red-Hat-Base</Category>
       <Not>
         <Or>
+          <Category>X-GNOME-PersonalSettings</Category>
+          <Category>DesktopSettings</Category>
+          <Category>X-GNOME-NetworkSettings</Category>
+          <Category>HardwareSettings</Category>
+          <Category>X-GNOME-SystemSettings</Category>
           <Category>System</Category>
-	  <Category>Accessibility</Category>
         </Or>
       </Not>
     </And>
   </Include>
 
+  <!-- Menu items to exclude from the toplevel -->
   <Exclude>
-      <Filename>redhat-userpasswd.desktop</Filename>
+    <Filename>gnomecc.desktop</Filename>
   </Exclude>
 
-  <!-- Accessibility -->
+  <!-- Personal submenu -->
   <Menu>
-    <Name>Accessibility</Name>
-    <Directory>Settings-Accessibility.directory</Directory>
+    <Name>Personal</Name>
+    <Directory>Personal.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>
-        <Category>Accessibility</Category>
-        <Not>
-          <Or>
-            <Category>System</Category>
-          </Or>
-        </Not>
+        <Category>X-GNOME-PersonalSettings</Category>
       </And>
     </Include>
-  </Menu>
+  </Menu> <!-- End Personal -->
 
-  <!-- More Prefs -->
+  <!-- Look and Feel submenu -->
+  <Menu>
+    <Name>Look and Feel</Name>
+    <Directory>LookAndFeel.directory</Directory>
+    <Include>
+      <And>
+        <Category>Settings</Category>
+        <Category>DesktopSettings</Category>
+      </And>
+    </Include>
+  </Menu> <!-- End Look and Feel -->
 
+  <!-- Internet and Network submenu -->
   <Menu>
-    <Name>MorePreferences</Name>
-    <Directory>Settings-More.directory</Directory>
+    <Name>Internet and Network</Name>
+    <Directory>InternetAndNetwork.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>
-        <Not>
-          <Or>
-            <Category>System</Category>
-            <Category>X-Red-Hat-Base-Only</Category>
-            <Category>X-Red-Hat-Base</Category>
-          </Or>
-        </Not>
+        <Category>X-GNOME-NetworkSettings</Category>
       </And>
-      <Filename>gnome-gconf-editor</Filename>
     </Include>
+  </Menu> <!-- End Internet and Network -->
   
-    <Exclude>
-      <Filename>redhat-userinfo.desktop</Filename>
-      <Filename>gnome-gdmphotosetup.desktop</Filename>
-    </Exclude>
+  <!-- Hardware submenu -->
+  <Menu>
+    <Name>Hardware</Name>
+    <Directory>Hardware.directory</Directory>
+    <Include>
+      <And>
+        <Category>Settings</Category>
+        <Category>HardwareSettings</Category>
+      </And>
+    </Include>
+  </Menu> <!-- End Hardware -->
 
-  </Menu>   <!-- End More Prefs -->
-</Menu>     <!-- End Prefs -->
+  <!-- System submenu -->
+  <Menu>
+    <Name>System</Name>
+    <Directory>System.directory</Directory>
+    <Include>
+      <And>
+        <Category>Settings</Category>
+        <Or>
+          <Category>X-GNOME-SystemSettings</Category>
+          <And>
+            <Category>System</Category>
+            <Not>
+              <Or>
+                <Category>X-GNOME-PersonalSettings</Category>
+                <Category>DesktopSettings</Category>
+                <Category>HardwareSettings</Category>
+                <Category>X-GNOME-NetworkSettings</Category> 
+              </Or>
+            </Not>
+          </And>
+        </Or>
+      </And>
+    </Include>
+  </Menu> <!-- End System -->
 
+</Menu>     <!-- End Prefs -->
